components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -87,7 +87,7 @@ const ContributorListTableRow = ({
 
   const repoList = useRepoList(Array.from(new Set(data.map((prData) => prData.full_name))).join(","));
   const contributorLanguageList = user ? getTopContributorLanguages(user) : [];
-  const days = getPullRequestsToDays(data, Number(range || "30"));
+  const days = getPullRequestsToDays(data, range ? Number(range) : 30);
```

Comment:
This is a quick fix to the last 30 days column being empty because the range was initially undefined.